static/directives/repo-list-table.html

```diff
@@ -62,6 +68,11 @@
          </span>
          <span class="empty" ng-if="::!repository.last_modified">(Empty Repository)</span>
        </td>
+       <td class="state" ng-show="repoMirroringEnabled">
```

Comment:
We use [these CSS classes for the background color](https://sourcegraph.com/github.com/quay/quay/-/blob/static/js/directives/ui/repo-state/repo-state.component.css#L62:6) in other places in the UI.